it/spring/boot-tomcat/src/main/java/com/linecorp/armeria/spring/tomcat/demo/SpringTomcatApplication.java

```diff
@@ -32,8 +31,7 @@
 
 @SpringBootApplication
 public class SpringTomcatApplication {
-
-    @Inject
+    @Autowired
```

Comment:
Is there any reason that you changed to this spring specific annotation instead of the annotation in JSR-330?

Reply:
+1 for using `@Inject` everywhere.

Reply:
For consistency. If `@Inject` is preferred, then I can change them all to `@Inject`.

Reply:
Yes, please.